CHANGELOG.md

```diff
@@ -23,6 +23,13 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian-labs/aws-resources/compare/release-1.15.0...master
 
+### Added
+- Add `CanonicalAmiProvider.Builder.avoidUnattendedUpgrades` for bumping `imageName` to Focal or higher.
+
+### Fixed
+- Unpin Ubuntu Focal version in `CanonicalAmiProvider.Builder` by default.
```

Comment:
We could rethink the approach if the less-repeatable builds around Ubuntu build would ever bite us. If we can easily switch back to the fixed build I think we are good to go.

Reply:
Nvm, via Java SDK it didn't work like "contains", but as "equals". Weird, because via GUI it always works like "contains". And on HTTP level the requests look almost identical (hostname is different).